pkg/cassandra/config/config.go

```diff
@@ -130,7 +131,13 @@ func (c *Configuration) NewCluster() *gocql.ClusterConfig {
 	} else {
 		cluster.Consistency = gocql.ParseConsistency(c.Consistency)
 	}
-	cluster.PoolConfig.HostSelectionPolicy = gocql.TokenAwareHostPolicy(gocql.RoundRobinHostPolicy())
+
+	fallbackHostSelectionPolicy := gocql.RoundRobinHostPolicy()
+	if c.LocalDC != "" {
+		fallbackHostSelectionPolicy = gocql.DCAwareRoundRobinPolicy(c.LocalDC)
+	}
+	cluster.PoolConfig.HostSelectionPolicy = gocql.TokenAwareHostPolicy(fallbackHostSelectionPolicy, gocql.ShuffleReplicas())
```

Comment:
what does `gocql.ShuffleReplicas()` do?

Reply:
There can be multiple replicas (equal to RF) for a given token. `ShuffleReplicas` selects one of these replicas at random instead of deterministically selecting a replica.